### PR TITLE
Add contracts for parser functions

### PR DESCRIPTION
**Pull request by tshemsedinov**

### CODE REVIEW

File: lib/parser.js [hunk 1]
```diff
@@ -1,41 +1,86 @@
...
+const readParcel = (buffer) => {
   // Read parcel from buffer
   // Return parcel object
   const rest = Buffer.from(buffer, 1, PARCEL_LENGTH);
   return rest;
 };
 
-const chunk = (buffer) => {
+const parcel = (
+  // Encode handshake structure to Buffer
```
Comment (o-rumiantsev) [inline lib/parser.js]: `Encode parcel structure` скорее

File: lib/parser.js [hunk 2]
```diff
@@ -1,41 +1,86 @@
 'use strict';
 
+const PROTOCOL_VERSION = 1;
```
Comment (mille-nium) [inline lib/parser.js]: In issue #3 2b for protocol version declared

Comment (tshemsedinov) [reply lib/parser.js]: We will use `writeInt16LE` to write this `Number` value to 2 bytes binary field @mille-nium

File: lib/parser.js (hunk 1 above)
Comment (tshemsedinov) [reply lib/parser.js]: Thanks

**Comment by tshemsedinov** [comment]
Landed in 4d26244

**Pull request closed without merging**